server/athenian/api/cache.py

```diff
@@ -111,6 +118,20 @@ async def wrapped_cached(*args, **kwargs):
                         .observe(len(payload))
             return result
 
+        async def reset_cache(*args, **kwargs) -> bool:
```

Comment:
@se7entyse7en This finally allows dropping a certain cache. It is not used anywhere except the tests yet.